.gitignore

```diff
@@ -4,4 +4,7 @@
 *.egg-info
 build
 /outputs
-/checkpoints
\ No newline at end of file
+/checkpoints
+__pycache__
+/dist
+/src
```

Comment:
Source code doesn't tend to be ignored..?

Reply:
Sorry I should have been clearer about this. The folder `src` does appear after installing dependencies and is not present otherwise in the repository. So I though that it should be ignored as it should already be present if it is something required. Am I wrong about this?